packages/dataviews/src/view-table.js

```diff
@@ -122,6 +139,7 @@ function HeaderMenu( { field, view, onChangeView } ) {
 					<DropdownMenuItem
 						prefix={ <Icon icon={ unseen } /> }
 						onClick={ () => {
+							onHide?.( field );
```

Comment:
`onHide` is always present, isn't it?

```suggestion
							onHide( field );
```

Reply:
Yes. In my head it was optional 🤷